client/cd-create-profile.c

```diff
@@ -757,6 +828,10 @@ cd_util_create_from_xml (CdUtilPrivate *priv,
 				     cd_dom_get_node_data (tmp));
 	}
 
+	ret = cd_util_adjust_creation_time (priv, filename, error);
```

Comment:
This could go on one line, i.e. `if (!cd_util_adjust_creation_time (priv, filename, error))`